lib/common/util.sh

```diff
@@ -314,3 +314,20 @@ unzip_file() {
     )
 }
 export -f unzip_file
+
+# delete files in directory that start like the given file name
+# (useful to purge error directory when same file is eventually
+# successfully processed)
+# $1 - directory
+# $2 - file name (basename)
+delete_files_in_dir() {
+    local dir="$1"; shift
+    local file="$1"; shift
+
+    log_info "Removing files in '$dir' that start with '$file'"
```

Comment:
Probably don't need to log this twice. On the other hand, if `$dir` does not exist, it would be helpful to log an error before returning.